core/src/lib/code-generation/custom-snippet.ts

```diff
@@ -113,6 +119,12 @@ export function initCustomSnippet(): CustomSnippets {
             codeBuilder: QueuedCodeBuilder,
             options: IPageObjectBuilderOptions,
         ) => {
+            const isCamelArrayId = Utils.isCamelArrayId.test(element.id);
```

Comment:
What about a Utils function like Utils.isArrayLike() instead of copying the same code twice?